src/collections/Articles.ts

```diff
@@ -0,0 +1,78 @@
+import type { CollectionConfig } from 'payload'
+
+export const Articles: CollectionConfig = {
+  slug: 'articles',
+  fields: [
+    {
+      name: 'title',
+      type: 'text',
+      required: true,
+    },
+    {
+      name: 'subtitle',
+      type: 'textarea',
+    },
+    {
+      name: 'content',
+      type: 'richText',
+      required: true,
+    },
+    {
+      name: 'author',
+      type: 'relationship',
+      relationTo: 'users',
+      required: true,
+      validate: async (value: any, { req }: { req: any }) => {
```

Comment:
No ```any``` types. Looks like value is a user id

Reply:
Not a problem anymore 😝